src/main/java/StringUtils.java

```diff
@@ -1,16 +1,66 @@
+/**
+ *
+ */
 public class StringUtils {
+    /**
+     * @param str //input of method
+     * @return boolean
+     * @throws Exception //throws WrongInputException or EmptyInputException
+     */
     public static boolean isPositiveNumber(String str) throws Exception {
-        if (str.length() < 1) //checks for blank input
+        isBlank(str);
+        boolean positive = isPositive(str);
+        int index = 0;
+        if (!positive){
+            index++;
+        }
+        containsCharacters(str, index);
+        return positive;
+    }
+
+    /**
+     * @param str //input
+     * @throws EmptyInputException //for blank input
+     */
+    //Checks for blank input
+    public static void isBlank(String str) throws EmptyInputException {
```

Comment:
Task description states that we need to have StringUtils class to have public isPositiveNumber method. 
Let's make methods private, so that not to expose inner methods by encapsulating other methods and follow task description.